x-pack/osquerybeat/ext/osquery-extension/pkg/browserhistory/browserhistory.go

```diff
@@ -41,23 +42,20 @@ func GetTableRows(ctx context.Context, queryContext table.QueryContext, log *log
 
 	results := make([]map[string]string, 0)
 
-	profileFilters := getConstraintFilters(queryContext, "profile_name")
-	userFilters := getConstraintFilters(queryContext, "user")
-	browserFilters := getConstraintFilters(queryContext, "browser")
-	filters := append(profileFilters, append(userFilters, browserFilters...)...)
+	allFilters := filters.GetConstraintFilters(queryContext)
 	locations, err := getSearchLocations(queryContext, log)
 	if err != nil {
 		return nil, err
 	}
 	var merr error
 	for _, location := range locations {
-		log.Infof("Parsing browser history for location: %v", location)
 		parsers := getParsers(ctx, location, log)
 		if len(parsers) == 0 {
 			continue
 		}
+		log.Infof("Parsing browser history for location: %#v", location)
```

Comment:
is there a standard for capitalization of log lines?  I've been starting mine with lower case.  Not sure if it matters.

Reply:
not really but something we can go through and make consistent